### ISSUE TITLE
Pypi Package

### ISSUE THREAD

**Issue by c4s4** [RittmanMead/md_to_conf#25]
It would be great to generate a Python package and make it available on Pypi.

**Comment by mineshpatel1** [comment]
I am no longer at RittmanMead, and where I currently am doesn't have Confluence so I can't test this very well, but I think this is a good idea if people find it useful.

**Comment by c4s4** [comment]
I can do it if you wish.

**Comment by mineshpatel1** [comment]
Go for it, the one thing I'd say to check is whether or not there is existing similar functionality that is better supported, perhaps by Atlassian themselves? I'm skeptical that this is still the best way of getting Markdown documents into Confluence.

**Comment by c4s4** [comment]
I'll check that, but I did it before and there was no good and free solution to import Markdown in Confluence at that time.

**Comment by stevegoossens** [comment]
I'd be interested in this too, as I'd like add SSL client cert auth ability for the Confluence REST connection, by just adding an extra argument (say, `--cert CERT_FILE`, along with corresponding env var `CONFLUENCE_CERT`) and have `request.Session` use it, like:

```
if USERNAME and API_KEY:
    session.auth = (USERNAME, API_KEY)
elif CERT:
    session.verify = CERT
else:
    LOGGER.error('Error: Credentials not specified by environment variable or option.')
    sys.exit(1)
```

I was going to make my own fork to add this, seeing as this repo appears abandoned, so if there's going to be a forked repo to make a PyPi package, I'd like to PR the client cert support to that repo (when I get round to making the change above and associated changes) :)

**Comment by mineshpatel1** [comment]
Yeah, I don't know if any of the RM folks are maintaining this. I was the original author of this, and have really no problem about forking this and letting someone else maintain it (I don't even have access to any Confluence instances, so I couldn't test anything anyway).

@borkur -> What do you reckon? Or are there folks at RM that still want to maintain this and build these features/review the pull requests (it looks like it has quite a few external users now). 

I did a quick Google and found this package: https://pypi.org/project/md2cf/. Don't know if that already serves your needs. Happy that people are still finding this little snippet useful.

**Comment by stevegoossens** [comment]
Ah yeah that `md2cf` looks pretty comprehensive. I'll have to check the rendering does what I expect/need, but the code appears to attach local images as attachments, so I'd probably just have to PR a tweak to their minimal Confluence client to support cert auth. Nice find! 😁
